package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "wp-autoupdates",
+	"version": "1.0.0",
```

Comment:
for completeness, shouldn't the name be something like "wordpress/wp-autoupdates"? and shouldn't the version match the WP_AUTO_UPDATES_VERSION in wp-autoupdates.php?

Reply:
I guess it depends on how long this repo is going to be around, if this is going to merged to core in the coming days...

Also feel free to a commit suggestion with the version number ;)

Reply:
I think technically, it would be `@wordpress/wp-autoupdates` to match this repo name. I'll update.